CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## main / unreleased
 
+* [ENHANCEMENT] Move rollout-operator libsonnet into this repository. Enable rollout-operator and its webhooks by default. #282
+* [CHANGE] Rename flag `enable_rollout_operator_webhook` to `rollout_operator_webhooks_enabled`. #282
+* [CHANGE] Add flag compatibility checks to prevent invalid flag combinations. #282
```

Comment:
Given the Jsonnet bundle wasn't in this repository before, I would describe this as an addition (rather than a move), and I don't think we need to list these as changes here.